run/healthchecks_liveness_probe_http/main.tf

```diff
@@ -26,37 +26,30 @@ resource "google_project_service" "cloudrun_api" {
 
 # Create Cloud Run Container with HTTP liveness probe
 #[START cloud_run_healthchecks_liveness_probe_http]
-resource "google_cloud_run_service" "default" {
+resource "google_cloud_run_v2_service" "default" {
   provider = google-beta
   name     = "cloudrun-service-healthcheck"
   location = "us-central1"
 
   template {
-    spec {
-      containers {
-        image = "us-docker.pkg.dev/cloudrun/container/hello"
+    containers {
+      image = "us-docker.pkg.dev/cloudrun/container/hello"
 
-        liveness_probe {
-          failure_threshold     = 5
-          initial_delay_seconds = 10
-          timeout_seconds       = 3
-          period_seconds        = 3
+      liveness_probe {
+        failure_threshold     = 5
+        initial_delay_seconds = 10
+        timeout_seconds       = 3
+        period_seconds        = 3
 
-          http_get {
-            path = "/"
-            http_headers {
-              name  = "Access-Control-Allow-Origin"
-              value = "*"
-            }
+        http_get {
+          path = "/"
+          http_headers {
```

Comment:
non-blocking: It might be nice to note why this header is needed.

Reply:
Added context